ui-components/src/locales/general.json

```diff
@@ -1221,8 +1221,8 @@
     "allApplicationClosed": "All applications are currently closed, but you can view closed listings.",
     "seeRentalListings": "See Rentals",
     "title": "Apply for affordable housing in",
-    "seeMoreOpportunities": "See more rental and ownership housing opportunities",
-    "viewAdditionalHousing": "View Additional Housing Opportunities and Resources"
+    "seeMoreOpportunities": "See which housing is right for you!",
+    "viewAdditionalHousing": "Check my eligibility"
```

Comment:
For strings that we want to be different in Detroit's instance, we override them in `sites/public/page_content/locale_overrides/general.json`.

Reply:
Actually, Chelsea can you create a new key for this string in this file, since the text is not related to "seeMoreOpportunities"?